DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/AccessoAllaPiattaforma.tex

```diff
@@ -1,58 +1,206 @@
 \UC{Registrazione}
-
 \begin{figure}[H]
     \centering
     \includegraphics[scale=0.6]{Immagini/DiagrammiUC/UC1.1Registrazione}
     \caption{Diagramma di \actualUC: Registrazione}
-    \label{fig:Registrazione}
+    \label{fig:RegistrazionePiattaforma}
 \end{figure}
 
 L'utente non autenticato che non dispone di credenziali può registrarsi e accedere come acquirente nella piattaforma usando l'email e la password.
 \begin{itemize}
-    \item \textbf{Attori Primari:} Utente non autenticato.
-    \item \textbf{Precondizione:} L'utente non è ancora presente nella piattaforma e si trova nella pagina di registrazione.
-    \item \textbf{Postcondizione:} L'utente è registrato con un account acquirente ed è autenticato come tale sulla piattaforma.
-    \item \textbf{Scenario Principale:} L'utente crea un account compilando tutti i campi del modulo di registrazione nel seguente modo:
+    \item \textbf{Attori primari:} utente non autenticato;
+    \item \textbf{Precondizione:} l'utente non è ancora presente nella piattaforma e si trova nella \glo{vista} di registrazione;
+    \item \textbf{Postcondizione:} l'utente è registrato con un account acquirente ed è autenticato come tale sulla piattaforma;
+    \item \textbf{Scenario principale:} l'utente crea un account compilando tutti i campi del modulo di registrazione nel seguente modo:
     \begin{itemize}
-        \item (UC23.1) - Inserimento indirizzo e-mail.
-        \item (UC23.2) - Inserimento password.
-        \item (UC23.3) - Inserimento conferma password.
+    	\item Inserimento nome;
+    	\item Inserimento cognome;
```

Comment:
Bisogna mettere gli scenari alternativi e le estensioni in caso non vengano messi il nome o il cognome. A questo punto farei dei sotto casi come per l'indirizzo email e la password